tests/unit/components/search-autocomplete/SearchAutocomplete.spec.jsx

```diff
@@ -1,13 +1,24 @@
-import { vi } from 'vitest'
-import { render, fireEvent, screen, waitFor } from '@testing-library/react'
+import { describe, vi } from 'vitest'
+import {
+  render,
+  fireEvent,
+  screen,
+  waitFor as _waitFor
```

Comment:
do you need to import it with the underline

Reply:
fixed